geode-core/src/distributedTest/java/org/apache/geode/internal/cache/partitioned/fixed/FixedPartitioningWithTransactionDistributedTest.java

```diff
@@ -238,7 +238,7 @@ private void forceClientMetadataUpdate(Region region) {
     ClientMetadataService clientMetadataService =
         ((InternalCache) clientCacheRule.getClientCache()).getClientMetadataService();
     clientMetadataService.scheduleGetPRMetaData((InternalRegion) region, true);
-    await().atMost(5, MINUTES).until(clientMetadataService::isMetadataStable);
+    await().atMost(5, HOURS).until(clientMetadataService::isMetadataStable);
```

Comment:
5 hours is a long time...Will this change going to be merged to develop? Can the debug be tried with local/private build?

Reply:
It is not reproducible in private/local build. We only need to have it to fail once, once the stack dump is triggered, we can/should change it back to default timeout.

Reply:
seems like this experiment could have been conducted in the PR pipeline, shouldn't have needed to merge to develop